LICENSE

```diff
@@ -0,0 +1,21 @@
+============================================
+PyInstaller community hooks: License details
+============================================
+
+This software is made available under the terms of the licenses
+found in LICENSE.APL.txt or LICENSE.GPL.txt Contributions to the community hooks are made
+under the terms of the license that covers that type of hook. (See below)
```

Comment:
I would reword this along these lines:
* Project is licensed under GPL-2.0-or later. (so this is the primary license)
  Thus this would cover also files which are not hooks ;-)
* As an exception for this, run-time hooks are licensed under Apache-2.0.

What do you think?

Reply:
For free Software, this file typically is called `COPYING`. WDYT?

Reply:
> For free Software, this file typically is called `COPYING`. WDYT?

https://stackoverflow.com/questions/5678462/should-i-provide-a-license-txt-or-copying-txt-file-in-my-project

It doesn't matter really. I prefer LICENSE; To me that says "These are the license terms with which you can use this code". But let me know if you'd like it changed.

I've reworded the file locally; I'll push the changes in a minute or two.

Reply:
In the main repo we call it "COPYING.txt", so haveing the same name here would be consistent. But as yu said, it actually dies not matter.
I have no strong preference, you decide.

Reply:
Re frst bullet-point: I'd put the GPL first, since this is the default :-)